File: Python_Assignments/Python_Assignment_5/Assignment5_1.py
#   1. Write a recursive program which display below pattern.

def Pattern(No):
    if No>0:
        print("*", end='\t')
        Pattern(No-1)

def main():
    Number = int(input("Enter your number : "))
    Pattern(Number)

if __name__ == "__main__":
    main()

#   Output:
#   D:\Python\Marvellous Python\Python Assignments\Python Assignment 5>Python Assignment5_1.py
#   Enter your number : 5
#   *       *       *       *       *

File: Logic_Building_Assignments/Logic_Building_Assignment_5/Assignment5_1.c
//  1. Write a program which accept number from user and print that number of
//     $&* on screen.

#include<stdio.h>

void Pattern(int iNo)
{
    int i = 0;

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    {
        for(i = 1; i <= iNo; i++)
        {
            printf("$\t*\t");
        }
    }
    
    printf("\n");
}

int main()
{
    int iValue = 0;

    printf("Enter a number : ");
    scanf("%d",&iValue);

    Pattern(iValue);
    return 0;
}

/*

    Output:
    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 5>gcc Assignment5_1.c

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 5>a
    Enter a number : 5
    $       *       $       *       $       *       $       *       $       *

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 5>a
    Enter a number : 3
    $       *       $       *       $       *

    D:\Marvellous Infosystems\Logic Building with Industrial Project Batch\Logic Building Assignments\Logic Building Assignment 5>a
    Enter a number : -3
    $       *       $       *       $       *

*/